components/button/button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Children } from "react";
+
+interface ButtonProperties {
+  children: React.ReactNode;
+  backgroundURL: string;
+  height: string;
+  width: string;
+  onClick: () => void;
+}
+
+const Button: React.FC<ButtonProperties> = ({ 
+  children, 
+  backgroundURL, 
+  height,
+  width,
+  onClick 
+}) => {
+  return (
+    <div className="container">
+      <button 
+        className="firstbutton-check" 
```

Comment:
Can we rename this className to something else? I'm not sure what firstbutton-check really means. Maybe we could aim for something along the lines on indicating it's the actually csutom button element like `custom-button`.